ch_pipeline/hfb/flagging.py

```diff
@@ -130,29 +130,29 @@ class HFBDirectionalRFIFlagging(task.SingleTask):
     """Produce a RFI mask based on HFB sensitivity values.
 
     This task detects RFI signals based on deviations from expected radiometer noise
-    in the second East-West beams (beam IDs: 256-512) for multiple values of std.
+    in the second East-West beams (beam IDs: 256-512) for multiple values of significance.
     It produces a directional RFI mask container recording the number of HFB subfrequency
-    channels flagged for RFI under each std.
+    channels flagged for RFI under each significance.
 
     Attributes
     ----------
     beam_ew_id : int
         The E-W beam index to inspect for RFI detection. Default is 1. (beam IDs: 256-512)
-    std : list of float
-        List of exactly four standard deviation values used as thresholds in RFI detection.
+    sigma : list of int
+        List of exactly four significance values used as thresholds in RFI detection.
         Each value corresponds to one of the four 8-bit segments packed into the 32-bit mask.
         These thresholds are used to compare against the sensitivity metric, and each one
         produces a separate mask encoding the number of HFB subfrequency channels flagged as RFI.
         The order of values determines their byte position: the first maps to bits 0-7, the second
-        to 8-15, and so on. Default is [0.25, 0.275, 0.30, 0.325].
-    sigma_threshold : float
+        to 8-15, and so on. Default is [4, 5, 6, 10].
+    std : float
         This is to detect RFI in HFB sensitivity values. If it exceeds
-        1 + treshold * std (or std_avg), then it indicates RFI events. Default is 5.
+        1 + sigma * std, then it indicates RFI events. Default is 0.25.
     """
 
     beam_ew_id = config.Property(proptype=int, default=1)
-    std = config.Property(proptype=list, default=[0.25, 0.275, 0.30, 0.325])
-    sigma_threshold = config.Property(proptype=float, default=5)
+    sigma = config.Property(proptype=list, default=[4, 5, 6, 10])
+    std = config.Property(proptype=float, default=0.25)
```

Comment:
is `std` a very descriptive name? "standard" could be anything. If it's related to standard-deviation then "standard" to me is the descriptive attribute of the "deviation".